lib/mongo/collection/view/readable.rb

```diff
@@ -24,6 +24,19 @@ class View
       # @since 2.0.0
       module Readable
 
+        # The count options potentially included in the options hash, using the
+        # configure method.
+        #
+        # @api private
+        COUNT_OPTIONS = ["hint", "limit", "max_time_ms", "skip", "comment"]
```

Comment:
The change overall needs a release note, and specifically for count options, passing `limit` and `skip` would change behavior meaningfully wouldn't it?

Reply:
yes it would, but you were already able to pass in limit and skip view the options, this just fixes it for the chained methods...

now that I think about it this change is pretty backwards breaking if you've been using these methods and didn't realize they weren't working... How would you like to handle that?

Reply:
Since we now have feature flags, let's put the change behind an opt-in feature flag.

Reply:
done!